run-unit-tests.sh

```diff
@@ -0,0 +1 @@
+stack exec tests $(stack path --local-install-root)/bin/examples-Test
```

Comment:
is this supposed to be an automated test? if so, why isn't this expressed in a more idiomatic way in cabal? note that cabal has native support to specify a test-suite that "depends" on another executable in the same package and knows how to invoke it... this here seems like a hacky way to achieve the same w/o cabal's support

Reply:
I did not realize it is possible to do it in idiomatic way. If you know how to do it, go ahead, I would have to go read docs.